test/resources/v5/members.test.js

```diff
@@ -213,5 +213,155 @@ describe('v5', () => {
         });
       });
     });
+
+    describe('cart', () => {
+      describe('#getShoppingCart()', () => {
+        const params = {
+          segment_code: 'status',
+        };
+        const apiCall = nock(RO.urls.getApiBaseUrl(), {
+          reqHeaders: {
+            Authorization: `Bearer ${ACCESS_TOKEN}`,
+          },
+        })
+          .get(`/programs/${programCode}/members/${memberUUID}/cart`)
+          .query(params)
+          .reply(200, {
+            status: 'OK',
+            result: {
```

Comment:
I don't think this exactly matches the actual response bodies.

Reply:
Ya, you're right 🙏 . I'll update those responses.